### PR TITLE
feat: Support for providing Platform token

### PR DESCRIPTION
**Pull request by bsenth200**
## What

What does this PR add or remove?

## Why

Why are these changes needed?

## How

How do these changes achieve the goal?

## Test

How has this been tested? How can a reviewer test it?

## Checklist

- [x] I have self-reviewed this PR
- [ ] I have added tests that prove the feature works or the fix is effective

### CODE REVIEW

File: core/sdk/src/framework/ripple_contract.rs
Comment (satlead) [inline core/sdk/src/framework/ripple_contract.rs:122]: Please use SessionAdjective::Platform for this contract

File: core/sdk/src/api/distributor/distributor_platform.rs
Comment (satlead) [inline core/sdk/src/api/distributor/distributor_platform.rs:56]:
```suggestion
          RippleContract::Session(SessionAdjective::Platform),
```

**Review by satlead — APPROVED** [review]
Wait for a QA approval as well before merging

**Review by satlead — APPROVED** [review]
lgtm

**Review by Nv-Delle1416 — APPROVED** [review]
QA has been completed on these changes - Approved

**Pull request merged**
